.github/pull_request_template.md

```diff
@@ -1,20 +1,20 @@
 
-<!-- If you are not submitting for a dataset, feel free to remove the content below  -->
-
+<!-- If you are submitting a dataset or a model for the model registry please use the corresponding checklists below otherwise feel free to remove them. -->
 
 <!-- add additional description, question etc. related to the new dataset -->
 
-## Checklist for adding MMTEB dataset
 
-<!-- 
-Before you commit here is a checklist you should complete before submitting
-if you are not 
- -->
-Reason for dataset addition:
-<!-- Add reason for adding dataset here. E.g. it covers task/language/domain previously not covered -->
+## General Checklist
+
+- [ ] Run tests locally to make sure nothing is broken using `make test`. 
+- [ ] Run the formatter to format the code using `make lint`. 
+
```

Comment:
Instead of adding all to the same template, could we create multiple templates under the `.github/PULL_REQUEST_TEMPLATE` directory? That way we can have `general.md`, `add_dataset.md`, etc.

Reference [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)
> To create multiple pull request templates and use the template query parameter to specify a template to fill the pull request body, type .github/PULL_REQUEST_TEMPLATE/, then the name of your pull request template. For example, .github/PULL_REQUEST_TEMPLATE/pull_request_template.md.

Reply:
I see what you mean now. It is indeed not a great experience. What we have now is good then.